gnovm/pkg/gnolang/preprocess.go

```diff
@@ -1423,7 +1424,16 @@ func preprocess1(store Store, ctx BlockNode, n Node) Node {
 							checkOrConvertType(store, last, n, &n.Args[0], ct, false)
 						}
 					default:
-						// do nothing
+						ctBase := BaseOf(ct)
```

Comment:
nitpick:

```suggestion
						ctBase := baseOf(ct)
```

Reply:
BaseOf just calls baseOf. What is the point?

Reply:
BaseOf as exported is used by other packages, and the two lines( see the line below) is written in different style, so I suggest jut baseOf(ct)